content/control-api/index.textile

```diff
@@ -0,0 +1,125 @@
+</a>
+
+h3(#create-access-token). Create an access token
+
+To create a new token use the following procedure click the "Create new access token" button. The "Create new access token" dialog is displayed:
+
+<a href="/images/screenshots/control-api/new-access-token.png" target="_blank">
+  <img src="/images/screenshots/control-api/new-access-token.png" style="width: 40%" alt="My Settings">
+</a>
+
+Then complete the information in the dialog according to your requirements:
+
+1. Enter a friendly name for your token, so you'll remember what it's used for.
+2. Select the capabilities you wish the token to have, depending on your use case.
+3. Click the "Create" button to create the token.
+
+h3(#using-access-token). Using the access token
+
+From the "My access tokens" area you can click the "Copy Token" button, to copy the button to the clipboard.
+
+You use the access token to validate your API requests. This is done by using the access token as a bearer token in the @Authorization@ of the HTTP request. For example, using Curl:
+
+```[sh]
+curl -H 'Authorization: Bearer <YOUR_ACCESS_TOKEN>' ...
+```
+
+h2(#ids). Account and application IDs
+
+Note that some requests require your Account ID, and some require your App ID.
+
+h3(#account-id). How to find your Account ID
+
+In the "Ably dashboard":https://ably.com/accounts/any, on the top menu bar, navigate to "Free account" (or other account name as appropriate) and select "Account settings": 
+
+<a href="/images/screenshots/control-api/account-settings-menu-item.png" target="_blank">
+  <img src="/images/screenshots/control-api/account-settings-menu-item.png" style="width: 40%" alt="Account Settings">
+</a>
+
+Your Account Settings are displayed.
+
+h3(#app-id). How to find your Application ID
+
+In the "Ably dashboard":https://ably.com/accounts/any click on the app you want to find the application ID for. Click on the "Settings" tab:
+
+<a href="/images/screenshots/control-api/application-settings.png" target="_blank">
+  <img src="/images/screenshots/control-api/application-settings.png" style="width: 70%" alt="Application Settings">
+</a>
+
+The App ID is displayed under "Application settings". It is also the first part of your API key for that application. For example, if your API key is @28AB6c.DEFi0Q@, then the App ID is @28AB6c@.
+
+h2(#quickstart). Quickstart
+
+To list all the apps associated with your account:
+
+```[sh]
+curl "https://control.ably.net/v1/accounts/{{ACCOUNT_ID}}/apps" \
+     -H 'Authorization: Bearer {{ACCESS_TOKEN}}' \
+     -H 'Accept: application/json'
+```
+
+h2(#rate-limits). Rate limits
+
+The Control API has certain limits on the number of API calls that can be made per second. The limits are as follows:
+
+|_. Authenticated |_. Limit (requests per hour) |_. Scope |
+| True | 4000 | Limit is per account |
+| False | 60 | Limit is by IP address |
+
+The limit at the token level is 2000. This prevents an integration from using up all the limits.
```

Comment:
2000 what? Is this the number of API calls per token? And what happens then? Do you need to generate a new token?

Reply:
I've corrected this section. 2000 calls per hour. If you exceed you shouldn't need a new access token, but you will get an error message.